sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -0,0 +1,1137 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# pylint:disable=too-many-lines,too-many-public-methods
+import base64
+import uuid
+from typing import Any, AsyncIterable, Optional, Iterable, List, Dict
+from functools import partial
+
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.polling import async_poller
+
+from azure.keyvault.certificates.models import(
+    AdministratorDetails,
+    CertificatePolicy,
+    CertificateOperation,
+    Certificate,
+    DeletedCertificate,
+    CertificateBase,
+    Contact,
+    Issuer,
+    IssuerBase,
+    KeyProperties,
+    SecretContentType,
+    LifetimeAction,
+    KeyUsageType
+)
+from ._polling_async import CreateCertificatePollerAsync
+from .._shared import AsyncKeyVaultClientBase
+
+
+class CertificateClient(AsyncKeyVaultClientBase):
+    """"CertificateClient defines a high level interface for
+    managing certificates in the specified vault.
+    Example:
+        .. literalinclude:: ../tests/test_examples_certificates_async.py
+            :start-after: [START create_certificate_client]
+            :end-before: [END create_certificate_client]
+            :language: python
+            :dedent: 4
+            :caption: Creates a new instance of the Certificate client
+    """
+
+    # pylint:disable=protected-access
+    @distributed_trace_async
+    async def create_certificate(
+        self,
+        name: str,
+        policy: Optional[CertificatePolicy] = None,
+        enabled: Optional[bool] = None,
+        tags: Optional[Dict[str, str]] = None,
+        **kwargs: "**Any"
+    ) -> CertificateOperation:
+        """Creates a new certificate.
+
+        If this is the first version, the certificate resource is created. This
+        operation requires the certificates/create permission.
+
+        :param name: The name of the certificate.
+        :type name: str
+        :param policy: The management policy for the certificate.
+        :type policy:
+         ~azure.keyvault.certificates.models.CertificatePolicy
+        :param enabled: Determines whether the object is enabled.
+        :type enabled: bool
+        :param tags: Application specific metadata in the form of key-value pairs.
+        :type tags: dict(str, str)
+        :returns: The created CertificateOperation
+        :rtype: ~azure.keyvault.certificates.models.CertificateOperation
```

Comment:
This returns a poller.

Reply:
changed the comment to a coroutine because the async polling doesn't actually return a poller, just a coroutine. Thanks!